pycaret/time_series/forecasting/oop.py

```diff
@@ -3586,6 +3644,23 @@ def plot_model(
             passed during setup. Pass these as key-value pairs. For available
             keys, refer to the `setup` documentation.
 
+        display_kwargs: dict, default = {} (empty dict)
```

Comment:
See if this can be somehow merged with `fig_kwargs` as that is used for all plotting related kwargs (for plotly)

Reply:
done